### ISSUE TITLE
Reaper Scans Double Page

### ISSUE THREAD

**Issue by vipzzx** [keiyoushi/tachiyomi-extensions-source#4470]
### Source information

Reaper Scans 1.4.53

### Source language

English

### Steps to reproduce

1. Download chapter
2. View chapter
3. Some pages will show up twice in a row 

### Expected behavior

Not have double pages

### Actual behavior

Showing same page in a row when downloaded

### Mihon/Tachiyomi version

J2K 1.7.4

### Android version

Android 14

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by Smol-Ame** [comment]
Check WebView to see if the source itself has duplicate pages, otherwise, come to Keiyoushi Discord repo-support channel for extra troubleshooting

**Comment by vipzzx** [comment]
sorry looks like it actually is a source issue. its a bit confusing since when i go to webview it is blocked by cloudflare
